demo-vm-setup/setup-kudu-demo-vm.sh

```diff
@@ -34,8 +38,19 @@ else
   fi
 fi
 
+# VM Settings default.
+: ${VM_NAME:=kudu-demo}
+: ${VM_NUM_CPUS:=2}
+: ${VM_MEM_MB:=6144}
+
+if ! which VBoxManage >/dev/null ; then
```

Comment:
I think it'd be better to perform this check before the (potentially costly) download of the image itself.